.github/workflows/post-merge-actions-dev.yml

```diff
@@ -0,0 +1,84 @@
+name: Development Node server build and deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+# Cannot use below code because container-signer stack not designed to be used in Dev    
```

Comment:
Shall we just remove this commented out code

Reply:
Done